src/app/Processor/ProcessorEdit/ConfigurationForm/CustomJsonSchemaBridge.tsx

```diff
@@ -80,6 +80,7 @@ export class CustomJsonSchemaBridge extends JSONSchemaBridge {
         name,
         label,
         type: "password",
+        autocomplete: "new-password",
```

Comment:
Why is this one `new-password` but the others `off`?

Reply:
Hi @manstis, I ended up removing this. 

As [the spec says](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#preventing_autofilling_with_autocompletenew-password), it helps with Firefox, making it suggest saved passwords instead of autofilling them. 

At the same time it introduces a new "issue" on Chrome, it makes the "strong password suggestion" drop down appear when filling out password fields.

Chrome is [way more used](https://gs.statcounter.com/browser-market-share/desktop/worldwide) than Firefox, so I opted to remove it.